## 01_mkrelease_autoconf213_debian by Junichi Uekawa <dancer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: change call to autoconf213 to autoconf2.13 which exists in Debian
## Updated for 3.7p3 

--- canna-3.7p3/mkrelease.sh~	2003-12-28 02:15:20.000000000 +0900
+++ canna-3.7p3/mkrelease.sh	2004-07-02 18:45:06.000000000 +0900
@@ -3,10 +3,10 @@
 set -e
 set -x
 cp Canna.conf.dist Canna.conf
-autoconf259
-autoheader259
+autoconf2.50
+autoheader2.50
 rm -rf autom4te.cache
 cd canuum
-autoconf213
-autoheader259
+autoconf2.13
+autoheader2.50
 rm -rf autom4te.cache
